Platform/zh-Hant.lproj/Localizable.strings

```diff
@@ -129,7 +129,7 @@
 "Allow Remote Connection" = "允許遠端連線";
 
 /* No comment provided by engineer. */
-"Allows passing through additional input from trackpads. Only supported on macOS 13+ guests." = "只支援 macOS 13 以上的客端系統，可以從觸控板傳遞其他輸入。";
+"Allows passing through additional input from trackpads. Only supported on macOS 13+ guests." = "只支援 macOS 13 以上的客體系統，可以從觸控板傳遞其他輸入。";
```

Comment:
My opinion: 👍 

“客端” is also an acceptable translation [^1]. However, I prefer your “客端” translation since it is more commonly used.

[^1]: https://wiki.centos.org/zh-tw/HowTos/Virtualization/VirtualBox/CentOSguest

Reply:
With all due respect, IMO, there are many terms on the referenced CentOS pages just using Traditional Chinese character, but the terms themselves are Simplified Chinese usages.